pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "django-model-import"
-version = "0.4.12"
+version = "0.5.0"
```

Comment:
No need to bump the minor version; you're not introducing a breaking change here.